HISTORY.md

```diff
@@ -1,5 +1,32 @@
 # Release History
 
+## 0.3.1 (2020-04-27)
+
+- Adds p2p_stub connection
+- Adds p2p_noise connection
+- Adds webhook connection
+- Upgrade error handling for error skill
```

Comment:
there's a mixture of third-person/second person

Reply:
probably better all in third person, since in the previous versions the majority of items was in third person ("this release does this and that")

Reply:
at line 8 it means "upgrades" instead of "upgrade"